src/materials/Material.js

```diff
@@ -294,6 +303,44 @@ class Material extends EventDispatcher {
 		data.stencilZFail = this.stencilZFail;
 		data.stencilZPass = this.stencilZPass;
 
+		if ( this.clippingPlanes !== null && Array.isArray( this.clippingPlanes ) ) {
```

Comment:
It's sufficient to write:
```js
if ( Array.isArray( this.clippingPlanes ) === true ) {
```
`isArray()` covers the `null` case.

Reply:
Ok, done. Are there any other reviews?